Bugzilla/Install.pm

```diff
@@ -330,35 +330,25 @@ sub create_admin {
     {
         say "\n" . get_text('install_admin_setup') . "\n";
     }
-
-    while (!$email) {
+    if (!$email) {
```

Comment:
I'm partial to either if (not $email) { or  unless ($email) {
as if (! is easy to read the wrong way.